test/acceptance/specs/WhatIsYourEmailSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.specs
+
+import acceptance.pages._
+import acceptance.utils._
+
+class WhatIsYourEmailSpec extends BaseSpec
+  with SpecHelper
+  with StubSave4Later
+  with StubAuthClient
+  with StubEmailVerification
+  with StubCustomsDataStore
+  with StubSubscriptionDisplay {
+
+  feature("Show 'What is your email' page when the email address is not verified") {
+
+    lazy val randomInternalId = generateRandomNumberString()
+    lazy val randomEoriNumber = "GB" + generateRandomNumberString()
+
+    scenario("Show 'What is your email' page when subscription display response email is not verified") {
```

Comment:
are we not defining the scenarios based on user behaviour, for eg. `Show 'What is your email' page when user's email is not verified` rather than using actual api name (`subscription display`) in scenarios ?

Reply:
updated the scenario description